.github/workflows/unit-tests.yml

```diff
@@ -72,12 +115,4 @@ jobs:
           TAB_API_KEY: ${{ secrets.TAB_API_KEY }}
           CI_COMMIT_SHA: ${{ github.event.pull_request.head.sha }}
           CI_PR_NUMBER: ${{ github.event.pull_request.number }}
-          CI_SUITE: unit
-
-      - name: Commit generated files
-        if: ${{ github.event_name != 'release' && github.event_name != 'schedule' }}
-        uses: KittyCAD/gha-actions-public/pr-commit@main
-        with:
-          app-id: ${{ secrets.MODELING_APP_GH_APP_ID }}
-          private-key: ${{ secrets.MODELING_APP_GH_APP_PRIVATE_KEY }}
-          commit-message: "Look at this (photo)Graph *in the voice of Nickelback*"
```

Comment:
I've never seen this run except in the case of a bad merge conflict. I don't expect these tests to be writing to disk.